src/python_ml_project_template/datasets/flow_dataset_pyg.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Optional, Protocol, Union, cast
+
+import rpad.partnet_mobility_utils.dataset as pmd
+import torch
+import torch_geometric.data as tgd
+from flowbot3d.datasets.flow_dataset import Flowbot3DDataset
+
+
+class Flowbot3DTGData(Protocol):
+    id: str  # Object ID.
+
+    pos: torch.Tensor  # Points in the point cloud.
+    flow: torch.Tensor  # instantaneous positive 3D flow.
+    mask: torch.Tensor  # Mask of the part of interest.
+
+
+class Flowbot3DPyGDataset(tgd.Dataset):
```

Comment:
any reason you can't import from flowbot3d directly? or is the interface different?

Reply:
Oh because now we have random seed here, in flowbot3d it has seed argument but it never passes things to it so it remains None all the time (which made the point sampling random)

Reply:
ooo good catch